DataCapturing.podspec

```diff
@@ -53,10 +55,15 @@ This framework can be included by your App if you are going to capture sensor da
   # So make sure tests are always located on the same level or below the podspec.
   s.test_spec 'Tests' do |test_spec|
     test_spec.source_files = 'DataCapturing/Tests/**/*.swift'
+    test_spec.resources = 'DataCapturing/Tests/**/*.sqlite'
```

Comment:
Adding some missing test resources. This should enable the tests when running from outside the IDE, which was a problem I have been fighting for years now.

Reply:
Maybe add some documentation about this fix then? Or else it could disappear during next refactoring?